src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -564,6 +571,15 @@ parseBinderNoParens = P.choice
                       , ParensInBinder <$> parens parseBinder
                       ] P.<?> "binder"
 
+parseBinderLocalDecl :: TokenParser Binder
```

Comment:
Would you consider this to be valid?

```purescript
case foo of
  _ -> ...
```

I admit it's unusual, but it might be useful (using `purescript-debug` for example), and I don't think it's worth fragmenting the parser for.